packages/brokers/src/brokers/redis/BaseRedis.ts

```diff
@@ -185,6 +194,7 @@ export abstract class BaseRedisBroker<TEvents extends Record<string, any>>
 	 * Destroys the broker, closing all connections
 	 */
 	public async destroy() {
+		await this.unsubscribe(Array.from(this.subscribedEvents));
```

Comment:
```suggestion
		await this.unsubscribe(...this.subscribedEvents);
```

think we don't really use `Array.from`.

Reply:
then it has to be 
`await this.unsubscribe([...this.subscribedEvents]);`
`unsubscribe` expecting an array 
pushed a commit

Reply:
Or we can change `unsubscribe` to spread the `args` for a nicer API IMO.

Reply:
Not a fan of spread.

> then it has to be
await this.unsubscribe([...this.subscribedEvents]);
unsubscribe expecting an array
pushed a commit

ah, yup.